cookbooks/chef_server/recipes/_push_server.rb

```diff
@@ -0,0 +1,4 @@
+
+chef_ingredient 'push-jobs-server' do
```

Comment:
I noticed this is installing opscode-push-jobs-server-2.1.0-1.el7.x86_64.rpm  -  that's bad right?